### PR TITLE
Added CodeChef Starter 6 (July)

### PR DESCRIPTION
**Pull request by RahulSurana123**
# Have you read the [Contributing Guidelines](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation/blob/master/CONTRIBUTING.md)?
- [x] YES
- [ ] NO

## Project URL
- 

## What is the current behavior?
- Issue Number: #...

## Category
- [ ] 𝑨𝑺𝑲 𝒀𝒐𝒖𝒓 𝑫𝒐𝒖𝒃𝒕𝒔
- [ ] 𝑴𝒊𝒔𝒄𝒆𝒍𝒍𝒂𝒏𝒆𝒐𝒖𝒔 𝑺𝒕𝒖𝒇𝒇.
- [ ] 𝑫𝑺𝑨
- [x] 𝑪𝒐𝒎𝒑𝒆𝒕𝒊𝒕𝒊𝒗𝒆 𝑷𝒓𝒐𝒈𝒓𝒂𝒎𝒎𝒊𝒏𝒈
- [ ] 𝑻𝒆𝒄𝒉𝒏𝒊𝒄𝒂𝒍 𝑺𝒖𝒃𝒋𝒆𝒄𝒕
- [ ] 𝑳𝒐𝒘 𝑳𝒆𝒗𝒆𝒍 𝑫𝒆𝒔𝒊𝒈𝒏
- [ ] 𝑷𝒓𝒐𝒋𝒆𝒄𝒕𝒔
- [ ] 𝑰𝒎𝒑𝒐𝒓𝒕𝒂𝒏𝒕 𝑩𝒐𝒐𝒌𝒔 𝒂𝒏𝒅 𝑹𝒆𝒔𝒐𝒖𝒓𝒄𝒆𝒔
- [ ] 𝑩𝒆𝒉𝒂𝒗𝒊𝒐𝒖𝒓𝒂𝒍 𝑰𝒏𝒕𝒆𝒓𝒗𝒊𝒆𝒘 𝑸𝒖𝒆𝒔𝒕𝒊𝒐𝒏𝒔

## Changes
- Description of changes

## **Screenshots** 
- (Add Runnig Work Demo Image)

## Checklist
- [x] I've read the [Contributing Guidelines](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation/blob/master/CONTRIBUTING.md).
- [x] I've checked the issue list before deciding what to submit.
- [x] I've follow [Tree of Index](https://github.com/AkashSingh3031/The-Complete-FAANG-Preparation#tree-of-index).
- [x] I've add work in proper folder
- [x] I've not delete/modify previous codes.

### CODE REVIEW

File: 3]. Competitive Programming/06]. CodeChef/2]. Compete/4]. Starters/July/Cricket Ranking.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+For each test case, output in a single line "A" (without quotes) if player A is better than player B and "B" (without quotes) otherwise.
+
+
+
+    ***********************************************************
+*/
+
+
+
```
Comment (AkashSingh3031) [inline 3]. Competitive Programming/06]. CodeChef/2]. Compete/4]. Starters/July/Cricket Ranking.cpp:31]: Check this code
And Also make a folder `Starters - 6` in place of month
like- `4]. Starters/01]. Starters - 6/<CODE_NAME>.cpp

**Review by AkashSingh3031 — CHANGES_REQUESTED** [review]
Check this code - 2 `Cricket Ranking`

And Also make a folder `Starters - 6` in place of month
like- `4]. Starters/01]. Starters - 6/<CODE_NAME>.cpp`

**Review by AkashSingh3031 — COMMENTED** [review]
Accepted

**Comment by RahulSurana123** [comment]
> Accepted

Thanks  please Merge it

**Pull request merged**